### PR TITLE
hreflang tags in head for SEO

### PR DESCRIPTION
**Pull request by cyberaleks**
For SEO  purpose it should be helpful to have hreflang tags  in head for each domain.
- Fixed resolving host name to boolean. Code line 71.
- Added Reflang links to head for SEO purpose.
e.g.
```html
<link rel="alternate" hreflang="x-default" href="https://example.com/">
<link rel="alternate" hreflang="de-DE" href="https://de.example.com/">
```

### CODE REVIEW

File: multiple-domain/multiple-domain.php [hunk 1]
```diff
@@ -207,8 +210,8 @@ public function settingsFields()
     {
         $fields = '';
         $i = 0;
-        foreach ($this->domains as $domain => $base) {
-            $fields .= $this->getDomainFields($i++, $domain, $base);
+        foreach ($this->domains as $domain => $values) {
+            $fields .= $this->getDomainFields($i++, $domain, $values['base'], $values['lang']);
```
Comment (straube) [inline multiple-domain/multiple-domain.php]: Hey @cyberaleks, it seems to me this part is not compatible with previous versions. Do you think we can handle this part to expect either a string (containing the optional base path) or an array (with the base path and lang code)?

Also, not everybody we'll use domains linked to different langs. I myself have a project that just uses a domain for part of the site due to some client requirements. I just wanted to make sure that lang is not required.

Thanks! :)

**Review by straube — CHANGES_REQUESTED** [review]
Hi! Thanks for your contribution. I think we just need to address some minor cases. Could you tackle them?

**Comment by cyberaleks** [comment]
Hi, i made lang optional and backwards compatible.

File: multiple-domain/multiple-domain.php (hunk 1 above)
Comment (cyberaleks) [reply multiple-domain/multiple-domain.php]: Fixed

**Comment by straube** [comment]
Hey @cyberaleks! Thanks for adding those changes! The last updates are now public: https://wordpress.org/plugins/multiple-domain/ :)

**Pull request merged**